examples/powertools-examples-core/cdk/app/pom.xml

```diff
@@ -0,0 +1,201 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
```

Comment:
What version of Java is the CDK forcing? It might be the case that we should only support Java > 1.8 here if the CDK requires that. @jeromevdl ?

Reply:
You can use different JDK versions for the CDK code and the function code...

Reply:
I'm happy to bump it to at least Java11, as the Java8 code in CDK is a bit clunky

Reply:
> I'm happy to bump it to at least Java11, as the Java8 code in CDK is a bit clunky



> You can use different JDK versions for the CDK code and the function code...

IMHO given CDK introduces a lower constraint on Java we may as well have the same on the example itself. Then for the build the whole CDK sub-tree only runs for Java-11 plus, or what have you. Otherwise it's almost academic - "yes, you can build the deployable component of this example, but you can't deploy it".

What do you think @jeromevdl ?

Reply:
> What version of Java is the CDK forcing? It might be the case that we should only support Java > 1.8 here if the CDK requires that. @jeromevdl ?

Answer [here](https://docs.aws.amazon.com/cdk/v2/guide/work-with-cdk-java.html): 

> Java AWS CDK applications require Java 8 (v1.8) or later. We recommend [Amazon Corretto](https://aws.amazon.com/corretto/), but you can use any OpenJDK distribution or [Oracle's JDK](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html). You will also need [Apache Maven](https://maven.apache.org/download.cgi) 3.5 or later. You can also use tools such as Gradle, but the application skeletons generated by the AWS CDK Toolkit are Maven projects.


So we actually have no excuse here to not support 1.8.

Reply:
I downgraded CDK examples to JDK8